### ISSUE TITLE
General Discussion 3

### ISSUE THREAD

**Issue by kierajreed** [olleharstedt/civ-clicker#30]
talk about more stuff

**Comment by kierajreed** [comment]
Olle, in one of your recent commits you mentioned help wanted for:
- [ ] Animal Sim
- [ ] Crop Sim
- [ ] Icons

I could do the icons and maybe a prototype crop sim.

**Comment by kierajreed** [comment]
you can delete the `hearth-fix` branch btw

**Comment by olleharstedt** [comment]
Hi!

Yeah, I'm thinking it would be good if we could get a designer on the team to remake the icons in use. They are not that fancy. ;) If you want to give it a go, feel free.

About crop simulator, sure. First step would be to find any information about it, like games that use it (did they blog about it) or academic papers.

**Comment by kierajreed** [comment]
True. Many of the icons were just cleverly cropped + edited images.

I've done a first-google and found this article, which is about rice growth depending on multiple factors:

- https://www.researchgate.net/profile/Grace_Centeno/publication/40190467_The_rice_simulation_model_SIMRIW_and_its_testing/links/55db360008aec156b9afe06c/The-rice-simulation-model-SIMRIW-and-its-testing.pdf

I have found other articles, but they were not available for free.

**Comment by olleharstedt** [comment]
Posted a blog post on reddit, got 70 upvotes! https://www.reddit.com/r/incremental_games/comments/79bq1y/civclicker_2_progress_report/

Also more subscribers to the civclicker subreddit, now 772: https://www.reddit.com/r/CivClicker/

**Comment by kierajreed** [comment]
Also, my save is making the Death Star, as a Small City.

Also, how about adding a clothing as a "tool" to decrease the get-sick temperature?

**Comment by kierajreed** [comment]
Also, can you tell me how you want the new icons to look?

**Comment by kierajreed** [comment]
How are the animal tests going?

**Comment by olleharstedt** [comment]
Reading up on the math, trying to find a suitable system.

**Comment by kierajreed** [comment]
I'm lost: what does `animal-test.js` do?
By the way, can you check out #29 please?

**Comment by olleharstedt** [comment]
Hi SkyHawkB!

animal-test.js belongs with animal-test.html. I'm trying out different equations I've found to see if I can fit them into the CivClicker setting. Check out this post: https://olleharstedt.github.io/civclicker/2017/10/28/adapting-predator-prey.html

**Comment by olleharstedt** [comment]
I'll check out #29 this weekend, busy with moving right now.

**Comment by kierajreed** [comment]
Hey, a new commit! What's it do?

**Comment by olleharstedt** [comment]
Sorry, I don't have time to explain every commit in detail.
